src/components/order-service/order-service.controller.ts

```diff
@@ -183,4 +190,14 @@ export class OrderServiceController implements angular.IController {
   private isNewProject(): boolean {
     return this.ctrl.selectedProject && !_.has(this.ctrl.selectedProject, 'metadata.uid');
   }
+  private watchResults = (resource: any, data: any, context: any) => {
+    this.watches.push(this.DataService.watchObject(resource, data.metadata.name, context, (instanceData: any, action: any) => {
+      var conditions: any = _.get(instanceData, 'status.conditions');
+
+      this.ctrl.orderComplete = _.find(conditions, {type: "Ready"});
+      var errorCondition: any = _.find(conditions, {status: "False"});
```

Comment:
@pmorie what's the current "perm failed" for provision, assuming thats the condition of type ProvisionedFailed that jeff is referencing?

@jeff-phillips-18 I doubt we have anything else to check for right now as far as conditions.  What you could have is an immediate failure when you make the provision request, but that would be in the error handler of the DataService.create